src/components/SignupFinish.js

```diff
@@ -0,0 +1,129 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { Fragment } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useCallback, useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import FormBuilder from 'components/Form/FormBuilder';
+import { formFieldTypes } from 'components/Form/CreateFormFields';
+
+import { Routes } from 'constants/Routes';
+import { routeWithParams } from 'lib/utils/routes';
+import { isValidEmail } from 'lib/utils/validations';
+
+import { useAuth } from 'hooks/useAuth';
+
+import PageLoader from 'components/Loader/PageLoader';
+import Note from 'components/Note';
+import Anchor, { anchorTypes } from 'components/Anchor';
```

Comment:
this is a nit - but can we organize imports by type?

external libs
internal libs
hooks
components
components in relative path (./blah)